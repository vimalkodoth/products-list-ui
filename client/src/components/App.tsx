/** @jsxImportSource @emotion/react */
import { AppCSS } from './App.styles';
import Products from './pages/Products';

export type TRows = string | number | boolean;

const App = (): JSX.Element => {
    return (
        <div css={AppCSS}>
            <Products />
        </div>
    );
};

export default App;
