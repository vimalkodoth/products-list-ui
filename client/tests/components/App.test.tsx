import React from 'react';
import { shallow } from 'enzyme';
import App from '../../src/components/App';

describe('App', () => {
    it('should render app component', async () => {
        const wrapper = shallow(<App />);
        expect(wrapper.getElements()).toMatchSnapshot();
    });
});
