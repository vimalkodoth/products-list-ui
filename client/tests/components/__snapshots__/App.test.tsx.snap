// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render app component 1`] = `
Array [
  <EmotionCssPropInternal
    __EMOTION_LABEL_PLEASE_DO_NOT_USE__="App"
    __EMOTION_TYPE_PLEASE_DO_NOT_USE__="div"
    css={
      Object {
        "map": undefined,
        "name": "1sax5zg",
        "next": undefined,
        "styles": "
    max-width: 720px;
    margin: 0px auto;
    height: 100vh;
",
        "toString": [Function],
      }
    }
  >
    <Products />
  </EmotionCssPropInternal>,
]
`;
